src/pyspextool/extract/extract_apertures.py

```diff
@@ -508,7 +508,7 @@ def write_apertures(spectra, psbginfo=None, xsbginfo=None, optimal_info=None,
                     plot_spectra(output_fullpath + '.fits',flag_linearity=True,
                                  file_info=qafileinfo)
 
-        elif extract.state['reductionmode'] == 'A-Sky/Dark':
```

Comment:
Why the change?  Sky/Dark is the most generic version?  That is, if we want people to use it, then this mode is meant for either a Sky or dark frame.